extensions/html-language-features/server/src/modes/javascriptMode.ts

```diff
@@ -41,7 +41,12 @@ function getLanguageServiceHost(scriptKind: ts.ScriptKind) {
 			getScriptSnapshot: (fileName: string) => {
 				let text = '';
 				if (fileName === currentTextDocument.uri) {
-					text = currentTextDocument.getText();
+					if (fileName.endsWith('html')) {
+						text = currentTextDocument.getText().replace(`<!--`, `/*`).replace(`-->`, `*/`);
```

Comment:
The replacement should have the same amount of chatacters, so that all offsets stay the same.
-> `/* ` instead of `/* `

Reply:
> Can you add a test case to
> 
> https://github.com/microsoft/vscode/blob/4b54ea927a64c9bd3e6d164edc7e021ca93dafcc/extensions/html-language-features/server/src/test/embedded.test.ts#L120
> ?

sure working on it

Reply:
Thanks, pleaese add also one with some JavaScript code before and after it.

Reply:
check the new commits, will that work?